packages/esm-form-entry-app/src/app/fe-wrapper/fe-wrapper.component.ts

```diff
@@ -90,13 +90,15 @@ export class FeWrapperComponent implements OnInit, OnDestroy {
       .subscribe(
         ({ form, concepts }) => {
           this.form = form;
-          this.labelMap = concepts.reduce((acc, current) => {
-            if (Boolean(current)) {
-              acc[current.identifier] = current.display;
-            }
+          if (concepts) {
+            this.labelMap = concepts?.reduce((acc, current) => {
```

Comment:
```suggestion
            this.labelMap = concepts.reduce((acc, current) => {
```

Reply:
Leads to this:

<img width="755" alt="Screenshot 2022-12-16 at 22 05 18" src="https://user-images.githubusercontent.com/8509731/208170655-4e582a82-5644-44eb-8190-97d8c0c31655.png">

Reply:
Even with the new `if` statement?

Reply:
Oh sorry, that was without the if